### PR TITLE
Added addresses as shared attribute

### PR DESCRIPTION
**Pull request by draganabukumira-verisec**

### CODE REVIEW

File: FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/SsnUserInfo.java [hunk 1]
```diff
@@ -12,20 +12,16 @@
...
+    private final Country country;
     private final String ssn;
 
-    public static SsnUserInfo create(Country country, String ssn) {
-        return new SsnUserInfo(country.getCountryCode(), ssn);
-    }
-
     @JsonCreator
-    private SsnUserInfo(@JsonProperty("country") String country, @JsonProperty("ssn") String ssn) {
+    public SsnUserInfo(@JsonProperty("country") Country country, @JsonProperty("ssn") String ssn) {
```
Comment (milicajelic-verisec) [inline FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/SsnUserInfo.java]: Why we set this public?

File: FrejaEidClient/src/test/java/com/verisec/frejaeid/client/beans/authentication/init/InitiateAuthetnicationRequestBuildersTest.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
 public class InitiateAuthetnicationRequestBuildersTest {
 
     private static final String EMAIL = "email";
-    private static final SsnUserInfo SSN_USER_INFO = SsnUserInfo.create(Country.SWEDEN, "123123123");
```
Comment (milicajelic-verisec) [inline FrejaEidClient/src/test/java/com/verisec/frejaeid/client/beans/authentication/init/InitiateAuthetnicationRequestBuildersTest.java]: I think we should still user create in tests.

File: FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/SsnUserInfo.java (hunk 1 above)
Comment (draganabukumira-verisec) [reply FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/SsnUserInfo.java]: Reverted. And added public static create(Country, String).

File: FrejaEidClient/src/test/java/com/verisec/frejaeid/client/beans/authentication/init/InitiateAuthetnicationRequestBuildersTest.java (hunk 2 above)
Comment (draganabukumira-verisec) [reply FrejaEidClient/src/test/java/com/verisec/frejaeid/client/beans/authentication/init/InitiateAuthetnicationRequestBuildersTest.java]: Fixed.

**Pull request merged**